source/manual/debian-packaging.html.md

```diff
@@ -130,8 +130,16 @@ be bumped up.
 Add an fpm recipe to packager and then use [the Jenkins
 job](https://ci.integration.publishing.service.gov.uk/job/build_fpm_package/) to
 create a Debian package. You can use the Vagrant development-VM to test
-the recipe prior to commiting to packager. You can copy this package to
-the aptly machine and then add the deb file to aptly.
+the recipe.
+
+#### Test the recipe
+
+- The Jenkins job will produce a `.deb` package in the `Build Artifacts`.
+- With Packager cloned to your local `govuk` folder, download your new package to the Packager root folder.
+- Start the VM, move to the Packager project and run `sudo dpkg -i ./your_package_name.deb`.
+- Ensure your package has been successful.
```

Comment:
```suggestion
- Ensure your package has been successfully installed.
```

Reply:
Thanks @thomasleese I've updated this in my latest commit